sonar-project.properties

```
@@ -0,0 +1,32 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_ocis-php-sdk
+sonar.projectVersion=1.6.0
```

```suggestion
sonar.projectVersion=0.1.0
```

I think that we want to start with some low version, until we actually start making releases.